src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -84,6 +84,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     request.getSession().setAttribute("user", username);
+    if(username.equals("lriffle")) {
+    	System.out.print("test");
```

Comment:
Would prefer using a logging library java.util.logging or if you're ambitious add LOG4J or something.